core/data.go

```diff
@@ -483,34 +486,80 @@ func (cb Bundles) FromEncodedBundles(eb EncodedBundles) (Bundles, error) {
 
 // PaymentMetadata represents the header information for a blob
 type PaymentMetadata struct {
-	// Existing fields
-	AccountID string
+	// AccountID is the ETH account address for the payer
+	AccountID string `json:"account_id"`
```

Comment:
Where is the json serialization used?

Reply:
It's used when we marshal/unmarshal the struct to/from json, i.e. in dataapi.
I've removed these for now